plugins/wallabag.koplugin/main.lua

```diff
@@ -0,0 +1,692 @@
+                end,
+            },
+            {
+                text = _("Go to download folder"),
+                callback = function()
+                    if FileManager.instance then
+--                        FileManager.instance:reinit( self.directory )
+                        FileManager:showFiles( self.directory )
+                    else
+                        FileManager:showFiles( self.directory )
+                    end
+                end,
+            },
+            {
+                text = "Settings",
+                callback_func = function()
+                    return nil
+                end,
+                sub_item_table = {
+                    {
+                        text = _("Configure Wallabag server"),
+                        keep_menu_open = true,
+                        callback = function()
+                            self:editServerSettings()
+                        end,
+                    },
+                    {
+                        text_func = function()
+                            local path
+                            if not self.directory or self.directory == "" then
+                                path = "Not set"
+                            else
+                                path = filemanagerutil.abbreviate(self.directory)
+                            end
+                            return _("Set download directory") .. " (" .. path .. ")"
+                        end,
+                        keep_menu_open = true,
+                        callback = function(touchmenu_instance)
+                            self:setDownloadDirectory(touchmenu_instance)
+                        end,
+                    },
+                    {
+                        text_func = function()
+                            local filter
+                            if not self.filter_tag or self.filter_tag == "" then
+                                filter = "All articles"
+                            else
+                                filter = self.filter_tag
+                            end
+                            return _("Filter articles by tag") .. " (" .. filter .. ")"
+                        end,
+                        keep_menu_open = true,
+                        callback = function(touchmenu_instance)
+                            self:setFilterTag(touchmenu_instance)
+                        end,
+                    },
+                    {
+                        text = _("Remotely delete Finished articles"),
+                        checked_func = function() return self.is_delete_finished end,
+                        callback = function()
+                            self.is_delete_finished = not self.is_delete_finished
+                            self:saveSettings()
+                        end,
+                    },
+                    {
+                        text = _("Remotely delete 100% read articles"),
```

Comment:
Or just finished?

Reply:
I want to make a distinction between articles that have been read to the end (100% progress) and articles that have been marked as finished/complete in the status screen.

That makes it possible to delete only the articles where the user took the extra step of setting the status, for example.